spikeinterface/preprocessing/zero_channel_pad.py

```diff
@@ -0,0 +1,87 @@
+from typing import Union
+import numpy as np
+from spikeinterface.core import BaseRecording, BaseRecordingSegment
+from .basepreprocessor import BasePreprocessor, BasePreprocessorSegment
+from spikeinterface.core.core_tools import define_function_from_class
+
+
+class ZeroChannelPaddedRecording(BaseRecording):
+    name = 'zero_channel_pad'
+    installed = True
+
+    def __init__(self, parent_recording: BaseRecording,
+                 num_channels: int, channel_mapping: Union[list, None] = None):
+        """Pads a recording with channels that contain only zero.
+
+        Parameters
+        ----------
+        parent_recording : BaseRecording
+            recording to zero-pad
+        num_channels : int
+            Total number of channels in the zero-channel-padded recording
```

Comment:
This is ambiguous : is it the total channel count or the added channel count

Reply:
Total! It's written